content/en/docs/File Format/implementationstatus.md

```diff
@@ -27,6 +28,11 @@ Implementations:
 
 ### Physical types
 
+Physical types are defined by the [`enum Type` parquet.thrift]
```

Comment:
I purposely chose not to make this a permalink so it always reflected the current version of the thrift file

This means over time the link may not point to the exact correct line, but I think people can find the relevant section using `union LogicalType` and we can update the links if needed